drake/examples/Quadrotor/QuadrotorControl.h

```diff
@@ -10,7 +10,9 @@ class QuadrotorControl {
     typedef drake::lcmt_quadrotor_control_t LCMMessageType;
     static std::string channel() { return "QUAD_CONTROL"; };
 
-    QuadrotorControl(void) : motors(Eigen::Vector4d::Zero()) {}
+    QuadrotorControl(void) : motors(Eigen::Vector4d::Zero()) {
+      motors << 1.226250, 1.226250, 1.226250, 1.226250;
```

Comment:
would it be cleaner to pass in the mass (or the quadrotor plant) and use mass/4 here?